lib/matplotlib/lines.py

```diff
@@ -617,7 +617,8 @@ def get_window_extent(self, renderer):
                                  ignore=True)
         # correct for marker size, if any
         if self._marker:
-            ms = (self._markersize / 72.0 * self.figure.dpi) * 0.5
+            extra_pts = self._markersize + self._markeredgewidth
```

Comment:
```suggestion
            extra_pts = self._markersize + self._markeredgewidth / 2
```

I think. The edge line is drawn centered on the marker outline. So only half of the width adds to the extent.

Reply:
You're right, but this is already accounted for in the `* 0.5` for both marker size and edge width in the next line. In the more general case, this multiplication is no longer needed, see newest commits.